frontend/src/metabase/ui/components/inputs/Select/Select.module.css

```diff
@@ -160,3 +160,7 @@
     text-transform: capitalize;
   }
 }
+
+.SelectClearButton {
+  color: var(--mb-color-text-dark) !important;
```

Comment:
Can we avoid the `!important` here?

Reply:
I couldn't think of a better way. Any suggestions? :pray: